ocs_ci/ocs/perftests.py

```diff
@@ -505,3 +509,140 @@ def get_time():
 
         """
         return time.strftime("%Y-%m-%dT%H:%M:%SGMT", time.gmtime())
+
+    def check_tests_results(self):
+        """
+        Check that all sub-tests (test multiplication by parameters) finished and
+        pushed the data to the ElastiSearch server.
+        It also generate the es link to push into the performance dashboard.
+        """
+
+        es_links = []
+        try:
+            input_file = open(self.results_file, "r")
```

Comment:
@Avilir Would it be more optimized if we use "with" to deal with files?

Reply:
this is the same IMO.

Reply:
using 'with' for file open will take care of closing the file operation also it handles the exceptions in-case of failures, so it's good to use 'with' statement for file handling.

Reply:
updated the function as requested.